pkg/operator/apiserver/controller/apiservice/apiservice_controller_test.go

```diff
@@ -147,6 +150,25 @@ func TestAvailableStatus(t *testing.T) {
 				}
 			},
 		},
+		{
+			name: "PreconditionsError",
+			preconditionsForEnabledAPIServices: func([]*apiregistrationv1.APIService) (bool, error) {
+				return false, fmt.Errorf("dummy error")
```

Comment:
What about coverage for `return true, fmt.Errorf("dummy error")`?

Reply:
https://github.com/openshift/library-go/pull/1694/files#diff-03f35e8de3821373388cc9e73d45c2c7ab8883cc85c911df416e41cca81c7350R161 ?

Reply:
Is there a test covering a case where `preconditionsForEnabledAPIServices` returns `true` _and_ a non-nil error?

Reply:
yes, all the other cases defined in this test, by default `preconditionsForEnabledAPIServices` is set to` func([]*apiregistrationv1.APIService) (bool, error) { return true, nil }` - https://github.com/openshift/library-go/blob/0cd17b55520a43287c67078fd984663e6d91e8a6/pkg/operator/apiserver/controller/apiservice/apiservice_controller_test.go#L200

Reply:
But the returned error value is nil there?

Reply:
ah, sorry, i misread your comment, okay, we can add one more case to this test.

Reply:
done.